docs/en/advanced_config/device_tracking.md

```diff
@@ -0,0 +1,68 @@
+# Device Tracking
```

Comment:
Device Tracking sounds like it could equally well be a "Find my drone" service. Can we find a better name?

I was thinking about something like "Asset Tracking & Logging"? We want it to be something instantly recognisable. Very open to suggestions.